hw/ip/csrng/doc/theory_of_operation.md

```diff
@@ -47,7 +47,7 @@ This process repeats until the `glen` field value has been matched.
 Because each request is pipelined, requests from other `cmd_stage` blocks can be processed before the original generate command is completely done.
 This provides some interleaving of commands since a generate command can be programmed to take a very long time.
 
-When sending an unsupported or illegal command, `CS_MAIN_SM_ALERT` will be triggered, but there will be no status response or indication of which app the error occurred in.
+When sending an unsupported or illegal command, `CMD_STAGE_INVALID_ACMD_ALERT` will be triggered, but there will be no status response or indication of which app the error occurred in.
```

Comment:
I think the status response is still sent out on the corresponding app interface...

Reply:
I renamed the alert because it doesn't come from the main SM anymore. I think maybe we can discuss this in person.